app/src/routes/v1/submission.ts

```diff
@@ -85,4 +85,13 @@ router.put(
   }
 );
 
+/** Updates is_deleted flag for a submission */
+router.patch(
+  '/:submissionId',
```

Comment:
Route is not specific enough for the actual functionality. This currently implies a generic patch function on a submission. A more appropriate route would be `/:submissionId/delete`